docs/api/alerting.asciidoc

```diff
@@ -0,0 +1,44 @@
+[[rules-api]]
+== Rules APIs
+
+The following APIs are available for managing {kib} rules.
+
+* <<rules-create-api, Create rule API>> to create a rule
+
+* <<rules-update-api, Update rule API>> to update the attributes for existing rules
+
+* <<rules-get-api, Get rule API>> to retrieve a single rule by ID
+
+* <<rules-delete-api, Delete rule API>> to permanently remove a rule
+
+* <<rules-find-api, Find rules API>> to retrieve a paginated set of rules by condition
+
+* <<alerting-list-rule-types-api, List alerting rule types API>> to retrieve a list of rule types
+
+* <<rules-enable-api, Enable rule API>> to enable a single rule by ID
+
+* <<rules-disable-api, Disable rule API>> to disable a single rule by ID
+
+* <<rules-mute-api, Mute alert API>> to mute alert for a single rule by ID
+
+* <<rules-unmute-api, Unmute alert API>> to unmute alert for a single rule by ID
+
+* <<rules-mute-all-api, Mute all alerts API>> to mute all alerts for a single rule by ID
+
+* <<rules-unmute-all-api, Unmute all alerts API>> to unmute all alerts for a single rule by ID
+
+* <<alerting-health-api, Get Alerting framework health API>> to retrieve the health of the Alerting framework
```

Comment:
The Actions API intro page has a link to the deprecated APIs. Should we do something similar here?

Reply:
Added in https://github.com/elastic/kibana/pull/94745/commits/d169b05e4b365271a474e8fd6d0babf141a229e6, but I noticed the legacy APIs don't render in the menu. I will investigate.

Reply:
@gchaps I'm not sure how the connector APIs managed to have a single page with all the deprecated APIs? https://www.elastic.co/guide/en/kibana/master/actions-and-connectors-legacy-apis.html. When I tried, it doesn't render or renders as individual menu items.

Should the deprecated alerting APIs be under a sub-menu or on a single page?

Reply:
I think the deprecated APIs work best on a single page.

Reply:
Per discussion with @gchaps I've added https://github.com/elastic/kibana/pull/94745/commits/1e49b599974f89034d9f5ef0412d1c959afaaae7 to fix the missing legacy APIs.

Reply:
@gchaps @lcawl thank you! I didn't know that trick works to add legacy APIs on a single page.